examples/produce_part_brep/partcad.yaml

```
@@ -0,0 +1,19 @@
+
+docs:
+  usage: |
+    ```shell
+    pc inspect box
+    ```
+
+parts:
+  box:
+    type: brep
+    desc: A box defined in BREP
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)